libraries/botbuilder-dialogs-adaptive/tests/LGGenerator.test.js

```diff
@@ -214,45 +244,45 @@ describe('LGLanguageGenerator', function() {
             assert.equal(result3, 'english-gb');
         });
 */
```

Comment:
Since we are indenting the start of the comment, we should update the indentation at the end as well.

Reply:
Updated in commit.